lib/service/Service.ts

```diff
@@ -34,17 +37,89 @@ class Service {
     this.lndClient = components.lndClient;
     this.raidenClient = components.raidenClient;
     this.pool = components.pool;
+    this.config = components.config;
     this.shutdown = components.shutdown;
 
     this.logger = Logger.rpc;
   }
 
   /**
-   * Placeholder for a method to return general information about an Exchange Union node.
+   * Get general information about this Exchange Union node.
    */
   public getInfo = async () => {
-    const lnd = await this.lndClient.getInfo();
-    return { lnd };
+    const info: any = {};
+
+    info.version = packageJson.version;
+
+    const pairs = await this.orderBook.getPairs();
+    info.peers = this.pool.peerCount;
+    info.pairs = pairs.length;
+
+    let peerOrdersCount: number = 0;
+    let ownOrdersCount: number = 0;
+    for (const key in pairs) {
+      const pair = pairs[key];
+
+      const [orders, ownOrders] = await Promise.all([
+        this.orderBook.getPeerOrders(pair.id, 0),
+        this.orderBook.getOwnOrders(pair.id, 0),
+      ]);
+
+      peerOrdersCount += orders.buyOrders.length + orders.sellOrders.length;
+      ownOrdersCount += ownOrders.buyOrders.length + ownOrders.sellOrders.length;
+    }
+
+    info.orders = {
+      peer: peerOrdersCount,
+      own: ownOrdersCount,
+    };
+
+    if (!this.config.lnd.disable) {
```

Comment:
I think it's better to do `this.lndClient.isDisabled()` here because it's possible that lnd was disabled for some other reason besides configuration - maybe it didn't load properly . Same for Raiden below.

Reply:
Are you sure about that? If `lnd` is enabled in the config you surely want that it is always connected to `xud`. I think `getInfo` should not only show some basic numbers but also the status of your node. And if the connection between `lnd` and `xud` gets interrupted for some reason there should be a simple way for the user to find out apart from the logs.